server/src/repository.rs

```diff
@@ -71,4 +132,82 @@ impl Users {
     pub async fn delete_user(&self, id: ObjectId) -> Result<DeleteResult, Error> {
         self.collection.delete_one(doc! { "_id": id }, None).await
     }
+
+    pub async fn first_user(&self, name: &str, password: &str) -> Result<(), CheeseError> {
```

Comment:
`first_user` is an unclear name for this. Maybe something more like `create_initial_user`. At least somehow indicating that this _creates_ an initial admin user.